specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -139,6 +139,29 @@ int main()
 }
 ```
 
+## Registering for App Notifications using assets
+
+For Unpackaged applications, the developer calls into Register API with DisplayName and Icon.
```

Comment:
`the developer can opt to call into`

The dev isn't forced to call into the new API (it is highly recommended though).